server/rest/annotation.py

```diff
@@ -160,6 +161,33 @@ def createAnnotation(self, item, params):
                     exc.args, ))
 
     @describeRoute(
+        Description('Copy an annotation from an item to an other.')
+        .param('annotationId', 'The ID of the annotation.', paramType='path',
+               required=True)
+        .param('itemDest', 'Pass this to move the annotation to this item.',
+               required=True)
+        .errorResponse('Write access was denied for the item.', 403)
+        .errorResponse('Validation Error: JSON doesn\'t follow schema.')
+    )
+    @access.user
+    @loadmodel(model='annotation', plugin='large_image')
+    @filtermodel(model='annotation', plugin='large_image')
+    def copyAnnotation(self, annotationId, itemDest, params):
+        user = self.getCurrentUser()
+        annotation = self.model('annotation', 'large_image').load(annotationId, region=params)
+        if annotation is not None:
+            itemSrc = self.model('item').load(annotation.get('itemId'), force=True)
```

Comment:
The `@loadmodel` decorator will assert that the annotation exists.  The conditional here isn't necessary.